tests/backup/inventory_test.js

```diff
@@ -259,7 +259,7 @@ Scenario(
     I.fillField(backupInventoryPage.fields.description, 'Test description');
     I.click(backupInventoryPage.buttons.addBackup);
     backupInventoryPage.verifyBackupSucceeded(backupName);
-    I.seeCssPropertiesOnElements(backupInventoryPage.elements.backupNameSpan(backupName), { 'text-overflow': 'ellipsis' });
```

Comment:
waitForVisible first

Reply:
the comment is for the removed line, the wait is in method in line 261